### PR TITLE
Php-Laravel Ably Agent Header

### LINKED ISSUE #14: Implement RSC7d (Ably-Agent header)
see https://github.com/ably/docs/pull/1034 and [spec point](https://ably-docs.herokuapp.com/client-lib-development-guide/features/#RSC7d) for context.

### PR DESCRIPTION
**Pull request by sacOO7**
- Fixes #14

### CODE REVIEW

File: src/AblyFactory.php [hunk 1]
```diff
@@ -34,8 +34,8 @@ public function make($clientOptions = null)
      */
     protected function createInstance($clientOptions)
     {
-        AblyRest::setLibraryFlavourString('laravel');
-
+        $laravelVersion = app()->version();
+        AblyRest::setLibraryFlavourString('laravel/'.$laravelVersion);
```
Comment (lmars) [inline src/AblyFactory.php:38]: I think here we should be using the new API (i.e. calling `setAblyAgentHeader`) and updating `composer.json` to require a version of `ably-php` which includes that new API (and if we haven't released a version with that change in it yet, then we should do that first).

Comment (sacOO7) [reply src/AblyFactory.php:38]: Yeah, I am planning to do it, once we have a new release ready 👍

Comment (sacOO7) [reply src/AblyFactory.php]: Done

**Review by lmars — APPROVED** [review]

**Review by ikbalkaya — APPROVED** [review]

**Review by owenpearson — APPROVED** [review]
LGTM, thanks

**Pull request merged**